modules/nf-core/blast/updateblastdb/tests/main.nf.test

```diff
@@ -0,0 +1,33 @@
+nextflow_process {
+
+    name "Test Process BLAST_UPDATEBLASTDB"
+    script "../main.nf"
+    process "BLAST_UPDATEBLASTDB"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "blast"
+    tag "blast/updateblastdb"
+
+    test("Should download a blast db folder from a db name") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [ [id:'mito'] ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.db },
+                { assert process.out.versions }
```

Comment:
Snapshots would be good here, as the files should in theory be stable. Certainly for the versions, if feasible also for the database.

Reply:
It is not possible. Since these are downloaded from NCBI https://ftp.ncbi.nlm.nih.gov/blast/db/ and neither the tool nor the FTP provide any version, it is not convenient nor practical to check it.

Reply:
Do any of the output files have a static name?

Reply:
Yep. They have. I will look for a test checking that.

Reply:
There are ways to check names, let me know if you need help